pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/bestpractices/xml/MissingOverride.xml

```diff
@@ -0,0 +1,461 @@
+
+    <test-code>
+        <description>Consider enum methods</description>
+        <expected-problems>0</expected-problems>
+        <code><![CDATA[
+            package net.sourceforge.pmd.lang.java.rule.bestpractices.missingoverride;
+
+            import net.sourceforge.pmd.lang.java.ast.ASTAnyTypeDeclaration;
+            import net.sourceforge.pmd.lang.metrics.Metric;
+            import net.sourceforge.pmd.lang.metrics.MetricKey;
+
+            public enum EnumWithInterfaces implements MetricKey<ASTAnyTypeDeclaration> {
+                Foo {
+                    @Override
+                    public Metric<ASTAnyTypeDeclaration> getCalculator() {
+                        return null;
+                    }
+                };
+
+                @Override
+                public Metric<ASTAnyTypeDeclaration> getCalculator() {
+                    return null;
+                }
+
+
+                @Override
+                public boolean supports(ASTAnyTypeDeclaration node) {
+                    return false;
+                }
+            }
+            ]]></code>
+    </test-code>
+
+    <test-code>
+        <description>Consider enum methods</description>
+        <expected-problems>2</expected-problems>
+        <expected-linenumbers>9,15</expected-linenumbers>
+        <code><![CDATA[
+            package net.sourceforge.pmd.lang.java.rule.bestpractices.missingoverride;
+
+            import net.sourceforge.pmd.lang.java.ast.ASTAnyTypeDeclaration;
+            import net.sourceforge.pmd.lang.metrics.Metric;
+            import net.sourceforge.pmd.lang.metrics.MetricKey;
+
+            public enum EnumWithInterfaces implements MetricKey<ASTAnyTypeDeclaration> {
+                Foo {
+                    public Metric<ASTAnyTypeDeclaration> getCalculator() {
+                        return null;
+                    }
+                };
+
+
+                public Metric<ASTAnyTypeDeclaration> getCalculator() {
+                    return null;
+                }
+            }
+            ]]></code>
+    </test-code>
+
+    <test-code>
+        <description>Consider methods with array parameters</description>
+        <expected-problems>1</expected-problems>
+        <expected-messages><!--FIXME should be arrayParams(String, int[], StringBuilder[]) - caused by #910 -->
+            <message>The method 'arrayParams(String, int, StringBuilder)' is missing an @Override annotation.</message>
+        </expected-messages>
+        <code><![CDATA[
+            package net.sourceforge.pmd.lang.java.rule.bestpractices.missingoverride;
+
+            import org.objectweb.asm.Label;
+
+            public class ConcreteClassArrayParams extends AbstractClass {
+
+                // missing
+                public void arrayParams(String dflt, int[] keys, StringBuilder[] labels) {
+                    super.arrayParams(dflt, keys, labels);
+                }
+            }
+        ]]></code>
+    </test-code>
+
+    <test-code>
```

Comment:
You can use the attribute `regressionTest="false"` on `<test-code>`, then this test case is skipped. The test case will appear in the stats as "skipped". And you can use the correct expected values. The comment, why this test is disabled, is still needed, though....

Reply:
Will do, thanks! The name of the attribute is maybe a bit misleading though